aneris/harmonize.py

```diff
@@ -433,7 +433,8 @@ def _postprocess_trajectories(self, scenario):
         # update variable name
         self._model = self._model.reset_index()
         self._model.sector = self._model.sector.str.replace(
-            self.suffix, 'Harmonized-DB')
+            self.suffix, 'Harmonized') # N.B. needs to be altered
```

Comment:
Agreed that this being hard-coded is undesirable. Could you instead add it to the configuration as an option? An example was how just recently the global trajectories was added. In short, you add it to the default configuration here: https://github.com/iiasa/aneris/blob/f615304a278c8033623eb81b5c9b4ab7a5151f86/aneris/_io.py#L18. And then pull it directly out of the `config` object, like was done here: https://github.com/iiasa/aneris/blob/f615304a278c8033623eb81b5c9b4ab7a5151f86/aneris/harmonize.py#L483.